kms-message/src/kms_kmip_tag_type_private.h

```diff
@@ -17,303 +17,325 @@
+   KMS_X (CertificateSubjectOU, 0x42010A)                                                   \
+   KMS_X (CertificateSubjectEmail, 0x42010B)                                                \
+   KMS_X (CertificateSubjectC, 0x42010C)                                                    \
+   KMS_X (CertificateSubjectST, 0x42010D)                                                   \
+   KMS_X (CertificateSubjectL, 0x42010E)                                                    \
+   KMS_X (CertificateSubjectUID, 0x42010F)                                                  \
+   KMS_X (CertificateSubjectSerialNumber, 0x420110)                                         \
+   KMS_X (CertificateSubjectTitle, 0x420111)                                                \
+   KMS_X (CertificateSubjectDC, 0x420112)                                                   \
+   KMS_X (CertificateSubjectDNQualifier, 0x420113)                                          \
+   KMS_X (CertificateIssuerCN, 0x420114)                                                    \
+   KMS_X (CertificateIssuerO, 0x420115)                                                     \
+   KMS_X (CertificateIssuerOU, 0x420116)                                                    \
+   KMS_X (CertificateIssuerEmail, 0x420117)                                                 \
+   KMS_X (CertificateIssuerC, 0x420118)                                                     \
+   KMS_X (CertificateIssuerST, 0x420119)                                                    \
+   KMS_X (CertificateIssuerL, 0x42011A)                                                     \
+   KMS_X (CertificateIssuerUID, 0x42011B)                                                   \
+   KMS_X (CertificateIssuerSerialNumber, 0x42011C)                                          \
+   KMS_X (CertificateIssuerTitle, 0x42011D)                                                 \
+   KMS_X (CertificateIssuerDC, 0x42011E)                                                    \
+   KMS_X (CertificateIssuerDNQualifier, 0x42011F)                                           \
+   KMS_X (Sensitive, 0x420120)                                                              \
+   KMS_X (AlwaysSensitive, 0x420121)                                                        \
+   KMS_X (Extractable, 0x420122)                                                            \
+   KMS_X (NeverExtractable, 0x420123)                                                       \
+   KMS_X_LAST (ReplaceExisting, 0x420124)
+
+/* Generate an enum with each tag value. */
+#define KMS_X(TAG, VAL) KMIP_TAG_##TAG = VAL,
+#define KMS_X_LAST(TAG, VAL) KMIP_TAG_##TAG = VAL
+typedef enum { KMS_XMACRO } kmip_tag_type_t;
+#undef KMS_X
+#undef KMS_X_LAST
+
+#define KMS_X(TAG, VAL) \
+   case KMIP_TAG_##TAG:          \
+      return #TAG;
+#define KMS_X_LAST KMS_X
+static KMS_MSG_INLINE const char *
+kmip_tag_to_string (kmip_tag_type_t tag)
+{
+   switch (tag) {
+   default:
+      return "Unknown tag";
```

Comment:
```suggestion
      return "Unknown KMIP tag";
```
Suggest more detailed message.